components/pkg-export-helm/Cargo.toml

```diff
@@ -19,6 +19,7 @@ habitat_pkg_export_docker = { path = "../pkg-export-docker" }
 habitat_pkg_export_kubernetes = { path = "../pkg-export-kubernetes" }
 # We need to lock here since v0.30.0 bumps to a version of pest that fails to build on Windows.
 handlebars = { version = "0.29.1", default-features = false }
+lazy_static = "*"
```

Comment:
I'd rather pin to a specific version, or at least use a less relaxed version constraint. The [README](https://github.com/rust-lang-nursery/lazy-static.rs#getting-started) suggests `=1.0`, and who am I to disagree?

Reply:
I just copy&pasted the line from other components and I'd like to be consistent with them.